perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -186,6 +187,9 @@ public class RegistrarManagerImpl implements RegistrarManager {
 	private static final String NAMESPACE_GROUP_HTML_MAIL_FOOTER = AttributesManager.NS_GROUP_ATTR_DEF;
 	static final String URN_GROUP_HTML_MAIL_FOOTER = NAMESPACE_GROUP_HTML_MAIL_FOOTER + ":" + FRIENDLY_NAME_GROUP_HTML_MAIL_FOOTER;
 
+	private static final String DEFAULT_GROUP_MSG_VO = "Your application to group %group_name% was automatically rejected," +
+		" because the VO application was rejected.";
```

Comment:
Maybe consider if the reject message could instead of VO have the name of the organization.